tsuru/resource_tsuru_cluster_test.go

```diff
@@ -0,0 +1,78 @@
+package tsuru
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	echo "github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/require"
+	"github.com/tsuru/go-tsuruclient/pkg/tsuru"
+)
+
+func TestAccTsuruCluster_basic(t *testing.T) {
+	fakeServer := echo.New()
+	fakeServer.POST("1.3/provisioner/clusters", func(c echo.Context) error {
+		// p := &tsuru.Cluster{}
```

Comment:
commented code :/

Reply:
I think we should have at least two basic tests: one with the least number of arguments passed, going through the default values and one using only custom values to create the resource...